app/views/your-psp/_worldpay-flex.njk

```diff
@@ -1,9 +1,17 @@
 <h2 class="govuk-heading-m govuk-!-margin-top-9">3DS Flex credentials</h2>
 
-<p class="govuk-body">Worldpay’s 3DS Flex improves how 3DS works and will also enable 3DS2 when required. There’s an additional fee to Worldpay to use 3DS Flex. <a class="govuk-link" href="https://www.worldpay.com/en-gb/products/3ds-flex">Find out more about 3DS Flex</a>.</p>
+<p class="govuk-body">Worldpay’s 3DS Flex improves how 3DS works and will also enable 3DS2 when required. There’s an additional fee to Worldpay to use 3DS Flex. <a class="govuk-link" href="https://www.payments.service.gov.uk/3d-secure-2/">Find out more about 3DS2 and 3DS Flex</a>.</p>
 
 <p class="govuk-body">If you have set up 3DS Flex on your Worldpay account, you need to enter the following details to turn 3DS Flex on. You’ll find these details in your <a class="govuk-link" href="https://secure.worldpay.com/sso/public/auth/login.html">Worldpay account</a>.</p>
 
+<div class="govuk-warning-text">
+  <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+  <strong class="govuk-warning-text__text">
+    <span class="govuk-warning-text__assistive">Warning</span>
+    If your credentials are wrong, you are not able to take payments.
+  </strong>
+</div>
```

Comment:
We should use the Nunjucks macro for this:

```suggestion
{{ govukWarningText({
  text: "If your credentials are wrong, you are not able to take payments.",
  iconFallbackText: "Warning"
}) }}
```

Reply:
Ah, fantastic! I did not know about that.